iaf/testing/README.md

```diff
@@ -0,0 +1,85 @@
+
+You can define the IBM Cloud credentials in the IBM provider block but it is recommended to pass them in as environment variables.
+
+Go [here](../../CREDENTIALS.md) for details.
+
+**NOTE**: These credentials are not required if running this Terraform code within an **IBM Cloud Schematics** workspace. They are automatically set from your account.
+
+## 2. Set OpenShift cluster environment variables
+
+This module test requires a running OpenShift cluster on IBM Cloud Classic with the basic MCM requirements. You need the cluster ID or name, and the resource group where it is running. These 2 parameters are passed to the testing code as environment variables. Execute this code:
+
+```bash
+export TF_VAR_cluster_id=<cluster_id>
+export TF_VAR_resource_group=<resource_group>
+```
+
+## 3. Set Cloud Pak Entitlement Key
+
+This module also requires an Entitlement Key. Obtain it [here](https://myibm.ibm.com/products-services/containerlibrary) and store it in the file `entitlement.key` in the root of this repository. If you use that filename, the file won't be published to GitHub if you accidentally push to GitHub. 
+
+## 4. Test
+
+### Using Terraform client
+
+Follow these instructions to test the Terraform Module manually
+
+Create the file `test.auto.tfvars` with the following input variables, these values are fake examples:
+
+```hcl
+on_vpc                       = "false"
+config_dir                   = ".kube/config"
+cluster_id                   = "btvlh6bd0di5v70fhqn0"
+entitled_registry_user_email = "John.Smith@ibm.com"
+resource_group               = "Default"
+```
+
+These parameters are:
+
+- `entitled_registry_user_email`: username or email address of the user owner of the entitlement key. There is no default value, so this variable is required.
+- `on_vpc`: Infrastructure where the cluster is running. The possible values are: `true` and `false`. The default value is `false`.
+- `resource_group`: Resource group where the cluster is running. Default value is `Default`
+- `config_dir`: Directory to download the kubeconfig file. Default value is `./.kube/config`
+- `cluster_id`: Cluster ID of the OpenShift cluster where to install IAF
+
+Execute the following Terraform commands:
+
+```bash
+terraform init
+terraform plan
+terraform apply -auto-approve
+```
+
+One of the Test Scenarios is to verify the YAML files rendered to install IAF, these files are generated in the directory `rendered_files`. Go to this directory to validate that they are generated correctly.
+
+## 5. Verify
+
+To verify installation on the Kubernetes cluster you need `kubectl`, then execute:
+
+```bash
+export KUBECONFIG=$(terraform output config_file_path)
+
+kubectl cluster-info
+
+# Namespace
+kubectl get namespaces $(terraform output namespace)
+
+# CatalogSource
+kubectl -n openshift-marketplace get catalogsource | grep IBM
+
+# Subscription
+kubectl -n openshift-operators get subscription | grep ibm-automation
```

Comment:
These subscriptions are showing up for me in the `iaf` project, not `openshift-operators`